<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>678</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 57, 102);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>251</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(9, 6, 109);
background-color: rgb(3, 57, 108);
background-color: rgb(4, 53, 95);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

background-color: rgb(2, 62, 84);
background-color: rgb(23, 10, 124);
background-color: rgb(7, 35, 72);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(26, 49, 127)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>781</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 57, 102);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="IDCustomer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(237, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameCustomer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(237, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmailCustomer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(237, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TelephoneCustomer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>571</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(237, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="HomeTell">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Home Phone</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="CellPhone">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>380</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Cell Phone</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RegisterButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>510</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Customer ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Telephone</string>
     </property>
    </widget>
    <widget class="QPushButton" name="QueryButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>510</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Query</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExcludeButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>510</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Exclude - ID</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
